content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -216,6 +212,12 @@ Apply the primary cluster's configuration.
 $ istioctl install -f istio-main-cluster.yaml --context=${MAIN_CLUSTER_CTX}
 {{< /text >}}
 
+If using the `istio-ingressgateway` option, expose Istiod, for example by using the provided sample configuration.
```

Comment:
this sentence seems like it's missing something. Maybe `you can expose istiod...`?

Also, maybe we should call it `the control plane` instead of `istiod`?